generators/client-2/templates/src/main/webapp/app/shared/_shared.ng2module.ts

```diff
@@ -24,12 +34,19 @@ import {
 @NgModule({
     imports: [
         <%=angular2AppName%>SharedLibsModule,
-        <%=angular2AppName%>SharedCommonModule,
+        <%=angular2AppName%>SharedCommonModule<%_ if (enableTranslation){ _%>,
+        TranslateModule.forRoot({
+          provide: TranslateLoader,
+          useFactory: (http: Http) => new TranslateStaticLoader(http, '/i18n', '.json'),
```

Comment:
doc says below

> By default, only the TranslateStaticLoader is available. It will search for files in i18n/*.json, if you want you can customize this behavior by changing the default prefix/suffix:

can you check